packages/shared/src/types/tsserver.ts

```diff
@@ -4,3 +4,4 @@ import { TsServerLocationSchema, TsServerDiagnosticSchema } from '../schemas/tss
 
 export type TsServerLocationType = z.infer<typeof TsServerLocationSchema>;
 export type TsServerDiagnosticType = z.infer<typeof TsServerDiagnosticSchema>;
+export type TsServerSuggestionType = TsServerDiagnosticType;
```

Comment:
Even though types are the same today, perhaps it would be best to infer this from the corresponding schema rather than the other one?

Reply:
Changed to independant schema and infer in case they every divulge